docs/book/src/SUMMARY.md

```diff
@@ -108,6 +108,7 @@
 
   - [Makefile Helpers](./reference/makefile-helpers.md)
   - [Project config](./reference/project-config.md)
+  - [Creating Events](./reference/creating-events.md)
```

Comment:
```suggestion
  - [Raising Events](./reference/creating-events.md)
```

Can you please add it after `[Using Finalizers](https://github.com/kubernetes-sigs/kubebuilder/blob/da10bf1678133baae9085a96aeffdb9a0dbaa12f/docs/book/src/reference/using-finalizers.md)`